packages/ra-ui-materialui/src/input/AutocompleteInput.stories.tsx

```diff
@@ -403,20 +403,7 @@ const BookEditWithReferenceAndCreationSupport = () => (
     >
         <SimpleForm>
             <ReferenceInput reference="authors" source="author">
-                <AutocompleteInput
-                    create={<CreateAuthor />}
-                    options={{
```

Comment:
I remove the code syntax error, but I don't know if the Idea was to render custom options here. Seems out of the scope of this story